docs/community-spotlight/index.rst

```diff
@@ -0,0 +1,16 @@
+===================
+Community Spotlight
+===================
+
+Welcome! These interviews are meant to be a resource for new documentarians looking to gain insight into the work of technical writing 
+by sharing the stories of our diverse community. Each interview tells the story of one community member and what they've learned over 
+the course of their career so far, along with providing some practical advice for new documentarians. 
+
```

Comment:
We should add a section about how people can get in touch with us if they want to participate? Might get more folks in the door.

Reply:
How do you think we should go about doing that? Should I put in my Slack handle or maybe give them an email address?

Reply:
You can just add a WTD email, and I can forward. `jobs@writethedocs.org`?